client/my-sites/domains/domain-management/contacts-privacy/card.jsx

```diff
@@ -135,27 +134,23 @@ class ContactsPrivacyCard extends React.Component {
 
 		return (
 			<div>
-				<SectionHeader label={ translate( 'Domain Contacts' ) } />
+				<Card className="contacts-privacy__card">
+					<p>{ translate( 'Your domain contact information' ) }</p>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **34** times:
`translate( 'Domain Contact Information' )` *ES Score: 7*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/domain-edit-contact-form-usability-fixes)